From 6a54124c89176d076636da9a4168b842c9383b7a Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Mon, 5 May 2014 10:07:35 -0400
Subject: [PATCH 024/117] fuse: use fusermount instead of guestunmount in error
 message

guestunmount does not exist in libguestfs 1.20.x.

Followup of 19bea5e464388cab00adf1b84199eb408deca4a4.
---
 src/fuse.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fuse.c b/src/fuse.c
index 1a31153..e65be13 100644
--- a/src/fuse.c
+++ b/src/fuse.c
@@ -1018,7 +1018,7 @@ guestfs__mount_local_run (guestfs_h *g)
   r = guestfs_exists (g, "/");
   guestfs_pop_error_handler (g);
   if (r == -1) {
-    error (g, _("you must call 'guestfs_mount' first to mount a filesystem on '/'.\nNote: '%s' is still mounted.  Use 'guestunmount %s' to clean up."),
+    error (g, _("you must call 'guestfs_mount' first to mount a filesystem on '/'.\nNote: '%s' is still mounted.  Use 'fusermount -u %s' to clean up."),
            g->localmountpoint, g->localmountpoint);
     return -1;
   }
-- 
2.5.0

